charts/zaakbrug/values.yaml

```diff
@@ -92,6 +92,14 @@ affinity: {}
 timeZone: Etc/UTC
 
 zaakbrug:
+  staging:
+    enabled: false
+    zakenApi:
+      rootUrl: http://zaakbrug-openzaak-nginx.zaakbrug/zaken/api/v1/
+    documentenApi:
+      rootUrl: http://zaakbrug-openzaak-nginx.zaakbrug/documenten/api/v1/
+    catalogiApi:
+      rootUrl: http://zaakbrug-openzaak-nginx.zaakbrug/catalogi/api/v1/
```

Comment:
Now it's the default which is: `{{zaakbrug+fullname}}-{{openzaak.fullname}}`

I could fill in the fullNameOverwrite. I'm not sure if it will also remove the prefix.